scripts/docker-server.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+if [ -z $REPO_PATH ]; then
+    echo "$0 ERROR: The REPO_PATH environment variable is required!"
+    echo "Try this: export REPO_PATH=\`pwd\`"
+    exit 5
+elif ! [ -d $REPO_PATH ]; then
+    echo "$0 ERROR: $REPO_PATH is not a valid directory"
+    exit 6
+fi
+
+docker run --interactive --mount type=bind,source=$REPO_PATH,target=/particle-iot/docs --net host --rm --tty particle/docs
```

Comment:
```bash
#!/bin/bash

# Consume parameters
shift

# Launch container with forwarded parameters
docker run --interactive --mount type=bind,source=$(pwd),target=/particle-iot/docs --net host --rm --tty particle/docs "$@"
```

Reply:
This solution _ASSUMES_ they are calling the script from the root of the `docs` repository, but would otherwise hide the docker ugliness and allow them to forward the calling options _(without needing to have `gnu-getopt` installed)_.